net/FlatBuffers/ByteBuffer.cs

```diff
@@ -391,15 +365,15 @@ protected ulong ReadLittleEndian(int offset, int count)
             {
                 for (int i = 0; i < count; i++)
                 {
-                  r |= (ulong)_buffer.Buffer[offset + i] << i * 8;
+                    r |= (ulong)_buffer.Buffer[offset + i] << i * 8;
                 }
             }
             else
             {
-              for (int i = 0; i < count; i++)
-              {
-                r |= (ulong)_buffer.Buffer[offset + count - 1 - i] << i * 8;
-              }
+                for (int i = 0; i < count; i++)
+                {
+                    r |= (ulong)_buffer.Buffer[offset + count - 1 - i] << i * 8;
+                }
             }
```

Comment:
Rather than reading byte by byte, couldn't this (and all the other methods like it) just use, e.g. BinaryPrimitives.ReadUInt64LittleEndian?

Reply:
It probably can, but I think that can be a separate PR. This was existing code that just got auto-formatted by VS.